ui/app/AppLayouts/Communities/panels/TagsPanel.qml

```diff
@@ -23,6 +23,7 @@ StatusScrollView {
     property var rightButtons: StatusButton {
         objectName: "confirmCommunityTagsButton"
         text: qsTr("Confirm Community Tags")
+        enabled: d.cntSelectedTags > 0
```

Comment:
nitpick, but `cntSelectedTags` makes it look like `cant` or `cunt` more than `count` to me hahaha. I'd rename it to `countSelectedTags`. It's just too more chars.

Reply:
Done